### PR TITLE
Make event-sources helm3 compliant

### LINKED ISSUE #8525: Conform event-sources charts with helm3
Refactor event-sources chart to be compliant with Helm3

**AC**
Improve kyma components charts by:
- [x] make charts compatible with helm 3
- [x] fix errors in schemas
- [ ] remove deprecated helm APIs
- [x] do not install resources in multiple namespaces within one chart
- [x] move custom resource definitions to the first step (cluster-essentials)
- [ ] enable parallel installation of charts by removing dependencies and waiting steps where possible
- [x] each kyma chart can be installed by `helm template | kubectl apply -f -`

### PR DESCRIPTION
**Pull request by nachtmaar**
**Description**

Changes proposed in this pull request:

- Migrate event-sources chart to Helm 3

**Related issue(s)**

See #8525

### CODE REVIEW

File: resources/event-sources/values.schema.json [hunk 1]
```diff
@@ -0,0 +1,211 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "title": "The root schema",
```
Comment (sayanh) [inline resources/event-sources/values.schema.json:5]: (nit): Lets put something meaningful. E.g.  `The event-sources chart schema`

Comment (nachtmaar) [inline resources/event-sources/values.schema.json:5]:
```suggestion
  "title": "The event-sources chart schema",
```

**Pull request merged**